suite/framework.yaml

```diff
@@ -6,15 +6,4 @@ keep_environments: Never # Options: Always, OnFail, Never
 logging:
     write_pod_logs: OnFail # Options: Always, OnFail
     # panic=5, fatal=4, error=3, warn=2, info=1, debug=0, trace=-1
-    level: 0
-
-# Specify the image and version of the chainlink image you want to run tests against. Leave blank for default.
-chainlink_image: public.ecr.aws/chainlink/chainlink
```

Comment:
How do we control which version and values we're working with? Only through ENV vals?

Reply:
Yes, why do we need a config and env vars at the same time?

Reply:
Also, I removed it from here because I'd like to have deployment vars and test harness vars decoupled, `chainlink_image` has nothing to do with a framework, not only CL node can be deployed.